napari_svg/layer_to_xml.py

```diff
@@ -110,6 +110,9 @@ def image_to_xml(data, meta):
         if NAPARI_GE_4_18:
             mapped_image = cmap.map(image)
         else:
+            # to keep backward comaptybility with napari before 0.4.18 
+            # to workaround bug in `vmap.map` we approximate colormap using 
+            # BINS_APPROX values
             colors = (cmap.map(np.linspace(0, 1, num=BINS_APROX)) * 255).astype(np.uint8)
```

Comment:
I still don't get what this is meant to do. It's not working on my machine with napari 0.4.17; if I run the tests, I still get the failure from https://github.com/napari/napari/pull/5805.

Reply:
After chatting a bit in private, turns our I must have done something wrong. All good here as well!